src/System.Web.OData/OData/Formatter/Serialization/ODataDeltaFeedSerializer.cs

```diff
@@ -59,8 +59,9 @@ public override void WriteObject(object graph, Type type, ODataMessageWriter mes
             Contract.Assert(feedType != null);
 
             IEdmEntityTypeReference entityType = GetResourceType(feedType).AsEntity();
+#pragma warning disable 0618
```

Comment:
Switch to use the latest codes. You can refer to the feature/netcore branch related changes.